src/sentry/models/groupinbox.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from enum import Enum
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, JSONField
+
+
+class GroupInboxReason(Enum):
+    NEW = 0
+    UNIGNORED = 1
+    REGRESSION = 2
+
+
+class GroupInbox(Model):
+    """
+    A Group that is in the inbox.
+    """
+
+    __core__ = False
+
+    group = FlexibleForeignKey("sentry.Group", unique=True)
+    reason = models.PositiveSmallIntegerField(null=False, default=GroupInboxReason.NEW.value)
+    reason_details = JSONField(null=True)
```

Comment:
This is fine, we should make sure to create a `jsonschema` for it so that we can at least control what data is in here.